ds_caselaw_editor_ui/templates/judgment/unhold.html

```diff
@@ -0,0 +1,31 @@
+{% extends "layouts/judgment_with_sidebar.html" %}
+{% block judgment_content %}
+  {% load i18n %}
+  <h2>
+    {% translate 'judgment.hold.unhold_title' %}
+    <br>
+    {{ context.judgment.name }}
+  </h2>
+
+  <form action="{% url 'unhold' %}" method="post">
+    {% csrf_token %}
+    <div class="judgment-component__actions">
+      <input
+        type="hidden"
+        name="judgment_uri"
+        value="{{ context.judgment.uri }}"
+      />
+      <input
+        type="hidden"
+        name="unhold"
+        value="false"
+      />
+      <input
+        class="button-cta"
+        type="submit"
+        value="{% translate 'judgment.unhold.unhold_button' %}"
```

Comment:
Super nitpicky, but could we tweak this to:
```suggestion
        value="{% translate 'judgment.hold.unhold_button' %}"
```
so it's more consistent with the pattern in publish?